storyboarding/scripting/compound_commands.rst

```diff
@@ -12,6 +12,405 @@ Loop
 .. Storyboard loops
 .. Remember that the numbers are based off 0!
 
-Trigger Conditions
-==================
+Trigger
+=======
 .. Originally called trigger-loops lol
+
+There is one mechanic within storyboarding that will allow you to create some sort of interactive effect with what is happening during gameplay: Triggers.
+Triggers can catch certain events and execute basic commands on your sprite in response.
+
+.. note:: Triggers are the most complex command currently provided by the storyboarding API and unfortunately they are accordingly buggy.
+  There is a whole batch of weird behaviour attached to the use of triggers that will be covered in its most common oddities throughout this chapter and at the end of this chapter in its less common oddities.
+
+
+Syntax
+------
+Like every other command, a trigger is always attached to a sprite or animation.
+
+Triggers consist of merely 4 elements:
+    - command-character
+    - trigger condition
+    - time window
+    - trigger group (optional)
+
+After the trigger command you have to list the commands that are to be executed when the trigger condition occurs.
+
+.. code-block:: c
+  :linenos:
+  :caption: An example of using a simple trigger.
+  :name: TriggerSyntax
+
+  Sprite,Background,Centre,"sb\aoba.png",192.625,140
+   F,0,23500,,0
+   S,0,23500,,0.2
+   T,HitSoundSoft,0,105330
+    F,0,0,,1
+    F,0,110,,0
+
+*T* is they character marking the triggercommand.
+
+*HitSoundSoft* is the trigger condition. You will learn about the different kinds of trigger conditions in the next chapter.
+
+*0,105330* marks start- and endtime of the trigger. There is no limit to how often a trigger can be executed within the specified timeframe.
```

Comment:
Followed your suggestion for wording. As for the inline coding, it would be consequent to do it for the other to fragments taken from the code as well.
I changed this to ``stuff`` for now as I'm mostly using that in the later parts as well.